webrtc.html

```diff
@@ -7189,9 +7189,9 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
         </table>
       </div>
       <section id="rtcdtlsfingerprint*">
```

Comment:
Any idea why this ID ends with a `*`? Compare with https://w3c.github.io/webrtc-pc/#rtciceserver-dictionary?

(Matters somewhat because we link to the sections from Blink's IDL files, and if this is a typo it'll presumably change.)